templates/tool_form.html

```
@@ -0,0 +1,50 @@
+        <form method="post">
+            {% for param, info in params.properties.items() %}
+            <div>
+                <label for="{{ param }}">{{ param }}</label>
+                {% if info.enum %}
+                <select name="{{ param }}" id="{{ param }}">
+                    {% for opt in info.enum %}
+                    <option value="{{ opt }}">{{ opt }}</option>
+                    {% endfor %}
+                </select>
+                {% elif info.type == 'array' %}
+                <textarea name="{{ param }}" id="{{ param }}" placeholder="Enter JSON array"></textarea>
```

HTMLHint (1.5.0)

[error] The id value [ {{ param }} ] must be unique.

(id-unique)